main.go

```diff
@@ -109,12 +112,22 @@ func main() {
 			metric.WithRequestDurationBucket(prometheus.ExponentialBuckets(0.05, 2, 8))))
 	service.WithLogger(goalogrus.New(log.Logger()))
 
-	// service.Use(metric.Recorder())
+	appDB := application.NewGormApplication(db)
+	tokenParser, err := token.NewManager(config)
```

Comment:
I would recommend using `token.DefaultManager(config)` instead. Then you can access the token manager later even if you don't have incoming request.

Reply:
yes, that's what I'll need anyways, so I can use the `Locate` method to extract the `sub` claim from the request token ;)